toolchain/sem_ir/ids.h

```diff
@@ -155,18 +159,25 @@ struct ConstantId : public IdBase, public Printable<ConstantId> {
   // logic here. LLVM should still optimize this.
   static constexpr auto Abs(int32_t i) -> int32_t { return i > 0 ? i : -i; }
 
-  // Returns the instruction that describes this constant value, or
+  // Returns the instruction that describes this template constant value, or
   // InstId::Invalid for a runtime value. This is not part of the public
   // interface of `ConstantId`. Use `ConstantValueStore::GetInstId` to get the
   // instruction ID of a `ConstantId`.
-  constexpr auto inst_id() const -> InstId {
-    CARBON_CHECK(is_valid());
-    return InstId(Abs(index) - IndexOffset);
+  constexpr auto template_inst_id() const -> InstId {
+    CARBON_CHECK(is_template());
+    return InstId(index);
+  }
+
+  // Returns the symbolic constant index that describes this symbolic constant
+  // value.  Requires is_symbolic. This is not part of the public interface of
```

Comment:
"This is not part of the public interface..." seems redundant, since it's in the `private` section. Am I misunderstanding something?

(Same goes for the previous function, although that's arguably a pre-existing problem)

Reply:
Removed.